charts/resoto/README.md

```diff
@@ -0,0 +1,72 @@
+# resoto
+
+![Version: 0.3.1](https://img.shields.io/badge/Version-0.3.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.3.0](https://img.shields.io/badge/AppVersion-2.3.0-informational?style=flat-square)
+
+A Helm chart for Kubernetes
+
+**Homepage:** <https://resoto.com>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| kushthedude |  |  |
+| aquamatthias | <eng@some.engineering> |  |
+
+## Source Code
+
+* <https://github.com/someengineering/resoto>
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| fullnameOverride | string | `""` |  |
+| image.pullPolicy | string | `"IfNotPresent"` |  |
+| image.repositoryFolder | string | `"ghcr.io/someengineering"` |  |
```

Comment:
```suggestion
| image.repositoryFolder | string | `"somecr.io/someengineering"` |  |
```
This should also be updated in `values.yaml`

Reply:
done @TheCatLady